src/gtk-3.20/scss/_global.scss

```diff
@@ -81,6 +81,7 @@ $toolbar_bg_color: $bg_color;
 $toolbar_fg_color: $fg_color;
 
 $titlebar_bg_color: $dark_bg_color;
+$titlebar_bg_uf_color: mix($bg_color, $dark_bg_color, .6);
```

Comment:
also i would like it to be named `titlebar_bg_unfocused` (as wm_bg, wm_border and wm_title above)